### PR TITLE
Results activity

### PR DESCRIPTION
**Pull request by KIsForCookie**

### CODE REVIEW

File: app/src/main/res/layout/activity_results.xml [hunk 1]
```diff
@@ -4,39 +4,151 @@
...
-        android:text="Results!"
+        android:background="@drawable/button_round_back"
+        android:drawableLeft="@drawable/ic_chevron_left_black"
+        android:onClick="openMainActivity"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <Button
-        android:id="@+id/game_button"
+    <TextView
```
Comment (yufeiyang-dev) [inline app/src/main/res/layout/activity_results.xml]: The "Result" title is no longer needed

File: app/src/main/res/layout/activity_results.xml [hunk 2]
```diff
@@ -4,39 +4,151 @@
...
+        android:layout_marginStart="16dp"
         android:layout_marginTop="16dp"
-        android:ems="10"
-        android:inputType="textPersonName"
-        android:text="Results!"
+        android:background="@drawable/button_round_back"
+        android:drawableLeft="@drawable/ic_chevron_left_black"
+        android:onClick="openMainActivity"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
```
Comment (Stevenfly) [inline app/src/main/res/layout/activity_results.xml]:
```suggestion:-10
    <Button
        android:id="@+id/main_menu_button"
        style="@style/button_round_back"
        android:onClick="openMainActivity"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent" />
```

Comment (yufeiyang-dev) [inline app/src/main/res/layout/activity_results.xml:29]:
```suggestion
<Button
    android:id=“@+id/main_menu_button”
    style=“@style/button_round_back”
    android:onClick=“openMainActivity”
    app:layout_constraintStart_toStartOf=“parent”
    app:layout_constraintTop_toTopOf=“parent” />
```

Comment (yufeiyang-dev) [reply app/src/main/res/layout/activity_results.xml:29]: from line 24 to line 34

**Review by yufeiyang-dev — CHANGES_REQUESTED** [review]
See other comments.

File: app/src/main/res/layout/activity_results.xml (hunk 2 above)
Comment (KIsForCookie) [reply app/src/main/res/layout/activity_results.xml:29]: I don't understand what you mean by this

Comment (KIsForCookie) [reply app/src/main/res/layout/activity_results.xml:29]: nvm I understand now

**Review by xjchong — APPROVED** [review]

**Review by yufeiyang-dev — APPROVED** [review]

**Pull request merged**